components/calendar/benches/date.rs

```diff
@@ -2,12 +2,24 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-mod fixtures;
+use serde::{Deserialize, Serialize};
```

Comment:
Suggestion: make the change smaller by keeping the fixture definitions in the separate file, and just not including them in the file where they are not used

Reply:
the problem is that multiple bench files include the same fixtures module, and some fixtures are unused in each benchmark. this puts the fixtures for each benchmark in the benchmark

Reply:
Oh I see, that's terrible...

I think well commented allow(dead_code) in a central file might be better